ansible_base/lib/utils/create_system_user.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from typing import Optional, Tuple
+
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.translation import gettext as _
+
+from ansible_base.lib.utils.settings import get_setting
+
+logger = logging.getLogger('ansible_base.lib.utils.create_system_user')
+
+"""
+These functions are in its own file because it is loaded during migrations so it has no access to models.
+"""
+
+
+def create_system_user(user_model=None) -> object:  # Note: We can't load models here so we can typecast to anything better than an object
+    #
+    # Creating the system user is a little tricky because we need to reference ourselves
+    #
+    if not user_model:
+        logger.error("You must pass a user_model to create_system_user")
+        return None
+
+    # First create a User object for the system user
+    system_user, created = user_model.objects.get_or_create(username=get_system_username()[0])
```

Comment:
No `defaults`? Don't have an email, or first_name?

Reply:
The only required field on User is username. So I left all of the defaults off. We could do defaults for the is_active but we have to apply other things (like set_unusable_password and created_by) anyway so I just made this `username`.